CHANGELOG-2.0.md

```diff
@@ -0,0 +1,13 @@
+# CHANGELOG for 2.0.x
+
+This changelog references the relevant changes done in 2.0 version.
+
+### 2.0.0(2018-03-26)
+
+* feature #24583 Adding a new debug:autowiring command (weaverryan)
+* This new version is only for PHP ⩾7.1.
+* Update all dependencies
+* DoctrineParamConverter is not used needed anymore
+* This bundle only convert hashid into an id
+* Replace `autowire` with `passthrough`.  
+This new parameter which permits to continue with next param converters available.
```

Comment:
whitespace between `+` and `This` ? or you didn't mean to set a new line ?

Reply:
No, it's intended to continue with the previous point.